p2p/protocol/identify/obsaddr.go

```diff
@@ -8,15 +8,32 @@ import (
 	ma "github.com/multiformats/go-multiaddr"
 )
 
+const ActivationTrsh = 4
+
 // ObservedAddr is an entry for an address reported by our peers.
 // We only use addresses that:
-// - have been observed more than once. (counter symmetric nats)
-// - have been observed recently (10min), because our position in the
+// - have been observed more than 3 times in last 10 min. (counter symmetric nats)
```

Comment:
I think the four observations should be able to occur over a longer timespan than ten minutes.